src/components/NavigationBar/NavigationBar.module.scss

```diff
@@ -3,6 +3,7 @@
 .navbar {
     background-color: maroon;
     color: white !important;
+    height: 60px;
```

Comment:
You are setting the height without any consideration to how that effects the padding. The padding is now overlapping the content making it look squished. You need to shrink the contents of the header bar (icons, text, logo) and left the header bar shrink to that content rather than explicitly setting the height.